core/src/main/java/org/hibernate/ogm/datastore/keyvalue/options/navigation/spi/BaseKeyValueStoreEntityContext.java

```diff
@@ -22,4 +24,13 @@
 	public BaseKeyValueStoreEntityContext(ConfigurationContext context) {
 		super( context );
 	}
+
+	public E mapStorage(MapStorageType mapStorage) {
```

Comment:
I am a bit reluctant about this as a common option for all K/V stores. Can we expose it only for Redis at this point where we really support it? We still can pull it up later on.